python_modules/dagster/dagster_tests/core_tests/test_instance.py

```diff
@@ -86,3 +91,13 @@ def noop_pipeline():
 
     assert run.execution_plan_snapshot_id == ep_snapshot_id
     assert run.execution_plan_snapshot_id == create_execution_plan_snapshot_id(ep_snapshot)
+
+
+@pytest.mark.parametrize('dirname', ('.', '..'))
+def test_dagster_home_raises(dirname):
+    os.environ['DAGSTER_HOME'] = dirname
```

Comment:
wow, im overriding the os.environ here for the session, use monkeypatch. Had to stare at it for hours before realizing